packages/web_pointer_interceptor/lib/src/web.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:html' as html;
+
+import 'package:flutter/widgets.dart';
+
+import 'shim/dart_ui.dart' as ui;
+
+const String _viewType = '__webPointerInterceptorViewType__';
+const String _debug = 'debug__';
+
+String _getViewType({bool debug = false}) {
+  return debug ? _viewType + _debug : _viewType;
+}
+
+void _registerWrapper({bool debug = false}) {
+  final String viewType = _getViewType(debug: debug);
+
+  ui.platformViewRegistry.registerViewFactory(viewType, (int viewId) {
+    final html.Element wrapper = html.DivElement();
```

Comment:
Is this sized correctly without any extra properties? I had expected something like `position: absolute; left: 0; top: 0; right: 0; bottom: 0`.

Reply:
IIRC the framework does this to the root Element returned from the factory:

* CK: https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart#L167
* DOM: https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/html/platform_view.dart#L68

In any case, I'm going to add this, because one of the things I want to change with the underlying platform views is making them *not* modify whatever the user returns, and instead position a widget they own internally (the future `slot`?).